import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java

```diff
@@ -68,6 +68,12 @@
   )
   String aetDictionaryPath = "";
 
+  @Parameter(
```

Comment:
I'd prefer to keep this simple and not potentially confuse users by having two flags for the same parameter. Can we just use the existing AET dictionary as a file for this test?

Reply:
Passing it this way also would also simplify gke/gce setup (no need to create configmap for gke/configure volume for gce), at least when AET dictionary is small enough.

Reply:
Ok